sdk/monitor/Azure.Monitor.OpenTelemetry/src/AzureMonitorOpenTelemetryOptions.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
```

Comment:
please no :)

Reply:
oh come on, we need to secure our jobs in the future 🤣

Reply:
This is bases on the feedback Josh give to us on exporter. The public API cannot be nullable at this point. I want to enable once the Azure SDK is ready for it.

Reply:
fair point.

Josh was going to follow up on this and see if we could still use this.